wasmplugin/rules/crs-setup.conf.example

```diff
@@ -398,21 +410,23 @@ SecDefaultAction "phase:2,log,auditlog,pass"
 # does not get evaluated if the request is being blocked early. So when you
 # disabled early blocking again at some point in the future, then new alerts
 # from phase 2 might pop up.
-SecAction \
-    "id:900120,\
-    phase:1,\
-    pass,\
-    t:none,\
-    nolog,\
-    setvar:tx.early_blocking=1"
+#SecAction \
+#    "id:900120,\
+#    phase:1,\
+#    pass,\
+#    t:none,\
+#    nolog,\
+#    tag:'OWASP_CRS',\
+#    ver:'OWASP_CRS/4.3.0',\
+#    setvar:tx.early_blocking=1"
```

Comment:
```suggestion
 SecAction \
     "id:900120,\
     phase:1,\
     pass,\
     t:none,\
     nolog,\
     tag:'OWASP_CRS',\
     ver:'OWASP_CRS/4.3.0',\
     setvar:tx.early_blocking=1"
```
Because of https://github.com/corazawaf/coraza-proxy-wasm?tab=readme-ov-file#multiphase, CRS default conf for coraza-proxy-wasm comes with early blocking enabled by default. Pretty sure this is also the cause of the failing CI: the failing tests are loading this CRS file config.